common/game_rules/000_wc_game_rules.txt

```diff
@@ -109,47 +109,67 @@ horde_invasion = {
 		desc = "RULE_HORDE_INVASION_STRENGTH_X0_5_DESC"
 	}
 }
-qiraji_invasion = {
-	name = "RULE_QIRAJI_INVASION_STRENGTH"
+# qiraji_invasion = {
+	# name = "RULE_QIRAJI_INVASION_STRENGTH"
+	# group = "RULE_GROUP_WARCRAFT"
+	# option = {
+		# name = default
+		# text = "RULE_QIRAJI_INVASION_STRENGTH_DEFAULT"
+		# desc = "RULE_QIRAJI_INVASION_STRENGTH_DEFAULT_DESC"
+	# }
+	# option = {
+		# name = x1_5
+		# text = "RULE_INVADERS_STRENGTH_X1_5"
+		# desc = "RULE_QIRAJI_INVASION_STRENGTH_X1_5_DESC"
+	# }
+	# option = {
+		# name = x2
+		# text = "RULE_INVADERS_STRENGTH_X2"
+		# desc = "RULE_QIRAJI_INVASION_STRENGTH_X2_DESC"
+	# }
+	# option = {
+		# name = x0_25
+		# text = "RULE_INVADERS_STRENGTH_X0_25"
+		# desc = "RULE_QIRAJI_INVASION_STRENGTH_X0_25_DESC"
+	# }
+	# option = {
+		# name = x0_5
+		# text = "RULE_INVADERS_STRENGTH_X0_5"
+		# desc = "RULE_QIRAJI_INVASION_STRENGTH_X0_5_DESC"
+	# }
+# }
+old_god_black_empire_invasion = {
+	name = "RULE_OLD_GOD_BLACK_EMPIRE_INVASION"
 	group = "RULE_GROUP_WARCRAFT"
 	option = {
-		name = default
-		text = "RULE_QIRAJI_INVASION_STRENGTH_DEFAULT"
-		desc = "RULE_QIRAJI_INVASION_STRENGTH_DEFAULT_DESC"
+		name = on
+		text = "RULE_OLD_GOD_BLACK_EMPIRE_INVASION_ON"
+		desc = "RULE_OLD_GOD_BLACK_EMPIRE_INVASION_ON_DESC"
 	}
 	option = {
 		name = x1_5
 		text = "RULE_INVADERS_STRENGTH_X1_5"
-		desc = "RULE_QIRAJI_INVASION_STRENGTH_X1_5_DESC"
+		desc = "RULE_OLD_GOD_INVASION_STRENGTH_X1_5_DESC"
 	}
 	option = {
 		name = x2
 		text = "RULE_INVADERS_STRENGTH_X2"
-		desc = "RULE_QIRAJI_INVASION_STRENGTH_X2_DESC"
+		desc = "RULE_OLD_GOD_INVASION_STRENGTH_X2_DESC"
+	}
+	option = {
+		name = off
+		text = "RULE_OLD_GOD_BLACK_EMPIRE_INVASION_OFF"
+		desc = "RULE_OLD_GOD_BLACK_EMPIRE_INVASION_OFF_DESC"
 	}
 	option = {
 		name = x0_25
 		text = "RULE_INVADERS_STRENGTH_X0_25"
-		desc = "RULE_QIRAJI_INVASION_STRENGTH_X0_25_DESC"
+		desc = "RULE_OLD_GOD_INVASION_STRENGTH_X0_25_DESC"
 	}
 	option = {
 		name = x0_5
 		text = "RULE_INVADERS_STRENGTH_X0_5"
-		desc = "RULE_QIRAJI_INVASION_STRENGTH_X0_5_DESC"
-	}
-}
-black_empire_invasion = {
```

Comment:
The same thing. The game rule is deleted and still used.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/87687110-657e0180-c796-11ea-9eeb-a9433c466a0b.png)

</details>

Reply:
Not fixed.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/87879539-79c33800-c9fc-11ea-9087-46a5c8360022.png)

</details>

Reply:
legit couldnt find it

Reply:
It's literally in the first event.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/88063370-03990f80-cb7b-11ea-849e-d7f2a65507d0.png)

</details>